app/controllers/request_controller.rb

```diff
@@ -167,6 +167,12 @@ def similar
   end
 
   def list
+    # respond with a 404 and do not execute the search if request was not for html
+    if request.format && !request.format.html?
+      respond_to { |format| format.any { head :not_found } }
+      return
+    end
```

Comment:
Wouldn't it be better to respond to the formats we do accept? https://github.com/mysociety/alaveteli/commit/1644cbffa6fff252fca85c407ec55e3dacd38a5f